api/src/controllers/cohort-session.js

```diff
@@ -62,22 +65,21 @@ router.post("/eligibility/2023/:id?", async (req, res) => {
   })(req, res);
 });
 
-router.get("/isInscriptionOpen/timeZoneOffset/:timeZoneOffset/:sessionName?", async (req, res) => {
+router.get("/isInscriptionOpen", async (req, res) => {
   const { error, value } = Joi.object({
     sessionName: Joi.string(),
     timeZoneOffset: Joi.number().required(),
   })
     .unknown()
-    .validate(req.params);
+    .validate(req.query, { stripUnknown: true });
 
   if (error) return res.status(400).send({ ok: false, code: ERRORS.INVALID_PARAMS });
   const { sessionName: cohortName, timeZoneOffset } = value;
 
-  const serverTimezoneOffsetInMilliseconds = new Date().getTimezoneOffset() * 60 * 1000; // Server's offset from UTC
   const userTimezoneOffsetInMilliseconds = timeZoneOffset * 60 * 1000; // User's offset from UTC
 
   // Adjust server's time for user's timezone
-  const adjustedTimeForUser = new Date().getTime() - serverTimezoneOffsetInMilliseconds + userTimezoneOffsetInMilliseconds;
+  const adjustedTimeForUser = new Date().getTime() - userTimezoneOffsetInMilliseconds;
```

Comment:
Est-ce que ça marche avec des timezones positives et négatives ?

Je me demande si un truc comme ça serait pas mieux
```suggestion
  const adjustedTimeForUser = new Date().getTime() + userTimezoneOffsetInMilliseconds*-1;
```